cfgov/v1/models/images.py

```diff
@@ -84,10 +84,19 @@ def resize(self, size):
         operation.run(mock_image, image=None, env={})
         return mock_image.width, mock_image.height
 
+    @property
```

Comment:
Ah, sorry. I thought you were commenting on the addition lower in this file. Yes, you're correct about what it does. It doesn't directly address the root issue being worked on, but I think it's still valuable to set that default value when inserting an image into rich text to the CFGOVImage alt value, rather than its title.

Do you see a downside to leaving it in?